run_corrMtrix.sh

```diff
@@ -0,0 +1,3 @@
+python produce_corrMatrix.py --obsName="mass4l"
+python produce_corrMatrix.py --obsName="mass4l" --bkg='zz'
+python produce_corrMatrix.py --obsName="pT4l"
```

Comment:
This script can be appended as a new step (step-8) in the script `RunEverything.py`.

```python
            if (args.step == 8):
                border_msg("Running corr matrix: " + obsName)

                command = 'python produce_corrMatrix.py --obsName="{obsName}"'.format(
                        obsName = obsName
                )
                logger.info("Command: {}".format(command))
                if (args.RunCommand): os.system(command)
```

Reply:
Hi @ram1123 , thanks for the review. I applied most of the changes, please have a look. Now sourcing the macro from python directory where most of macros exist.
read_bins.py script is bit modified, which can accept the 2D bins from current format in current [observables_list.yml](https://github.com/vukasinmilosevic/Fiducial_XS/blob/CMSSW_10_X/Inputs/observables_list.yml) macro from `CMSSW_10_X` branch (can be reverted if do not like the idea).
 Let me know if there are additional comments.

Reply:
@ram1123 , I am updating the `yml` file with new binning for few observables. Could you please have a look on the update `read_bins.py` module ?
In the current format of [observables_list.yml](https://github.com/vukasinmilosevic/Fiducial_XS/blob/CMSSW_10_X/Inputs/observables_list.yml) does not support for few 2D observables for paring. 

Do you want to change the format in observables_list.yml or want to switch to module which I updated in this PR and can accept current format (as per AN table) ?